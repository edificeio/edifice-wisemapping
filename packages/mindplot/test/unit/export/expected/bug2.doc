<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Word.Document"?><w:wordDocument xmlns:sl="http://schemas.microsoft.com/schemaLibrary/2003/core" xmlns:w="http://schemas.microsoft.com/office/word/2003/wordml" xmlns:w10="urn:schemas-microsoft-com:office:word" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882" xmlns:wx="http://schemas.microsoft.com/office/word/2003/auxHint" xmlns:aml="http://schemas.microsoft.com/aml/2001/core" xmlns:o="urn:schemas-microsoft-com:office:office">
<o:DocumentProperties>
<o:Title>SaberMás</o:Title>
</o:DocumentProperties>
<w:styles>
<w:versionOfBuiltInStylenames w:val="4"/>
<w:latentStyles w:defLockedState="off" w:latentStyleCount="156"/>
<w:style w:type="paragraph" w:default="on" w:styleId="Normal">
<w:name w:val="Normal"/>
<w:rsid w:val="00831C9D"/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:sz w:val="24"/>
<w:sz-cs w:val="24"/>
<w:lang w:val="EN-US" w:fareast="EN-US" w:bidi="AR-SA"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading1">
<w:name w:val="heading 1"/>
<wx:uiName wx:val="Heading 1"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading1"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="32"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading2">
<w:name w:val="heading 2"/>
<wx:uiName wx:val="Heading 2"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading2"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="1"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading3">
<w:name w:val="heading 3"/>
<wx:uiName wx:val="Heading 3"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading3"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="2"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading4">
<w:name w:val="heading 4"/>
<wx:uiName wx:val="Heading 4"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading4"/>
<w:keepNext/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="3"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="28"/>
<w:sz-cs w:val="28"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading5">
<w:name w:val="heading 5"/>
<wx:uiName wx:val="Heading 5"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading5"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="4"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:i/>
<w:i-cs/>
<w:sz w:val="26"/>
<w:sz-cs w:val="26"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading6">
<w:name w:val="heading 6"/>
<wx:uiName wx:val="Heading 6"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading6"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="5"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:b/>
<w:b-cs/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading7">
<w:name w:val="heading 7"/>
<wx:uiName wx:val="Heading 7"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading7"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="6"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading8">
<w:name w:val="heading 8"/>
<wx:uiName wx:val="Heading 8"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading8"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="7"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
<w:i/>
<w:i-cs/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="Heading9">
<w:name w:val="heading 9"/>
<wx:uiName wx:val="Heading 9"/>
<w:basedOn w:val="Normal"/>
<w:next w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Heading9"/>
<w:spacing w:before="240" w:after="60"/>
<w:outlineLvl w:val="8"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:sz w:val="22"/>
<w:sz-cs w:val="22"/>
</w:rPr>
</w:style>
<w:style w:type="character" w:default="on" w:styleId="DefaultParagraphFont">
<w:name w:val="Default Paragraph Font"/>
<w:semiHidden/>
</w:style>
<w:style w:type="table" w:default="on" w:styleId="TableNormal">
<w:name w:val="Normal Table"/>
<wx:uiName wx:val="Table Normal"/>
<w:semiHidden/>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
<w:tblPr>
<w:tblInd w:w="0" w:type="dxa"/>
<w:tblCellMar>
<w:top w:w="0" w:type="dxa"/>
<w:left w:w="108" w:type="dxa"/>
<w:bottom w:w="0" w:type="dxa"/>
<w:right w:w="108" w:type="dxa"/>
</w:tblCellMar>
</w:tblPr>
</w:style>
<w:style w:type="list" w:default="on" w:styleId="NoList">
<w:name w:val="No List"/>
<w:semiHidden/>
</w:style>
<w:style w:type="paragraph" w:styleId="Title">
<w:name w:val="Title"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="Title"/>
<w:spacing w:before="240" w:after="60"/>
<w:jc w:val="center"/>
<w:outlineLvl w:val="0"/>
</w:pPr>
<w:rPr>
<w:rFonts w:ascii="Arial" w:h-ansi="Arial" w:cs="Arial"/>
<wx:font wx:val="Arial"/>
<w:b/>
<w:b-cs/>
<w:kern w:val="28"/>
<w:sz w:val="32"/>
<w:sz-cs w:val="32"/>
</w:rPr>
</w:style>
<w:style w:type="paragraph" w:styleId="BodyText">
<w:name w:val="Body Text"/>
<w:basedOn w:val="Normal"/>
<w:rsid w:val="00BA7540"/>
<w:pPr>
<w:pStyle w:val="BodyText"/>
<w:spacing w:after="120"/>
</w:pPr>
<w:rPr>
<wx:font wx:val="Times New Roman"/>
</w:rPr>
</w:style>
</w:styles>
<w:body>
<wx:sect>
    <wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Title"/>
</w:pPr>
<w:r>
<w:t>SaberMás</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Complementamos el trabajo de la escuela</w:t>
</w:r>
</w:p>
<w:p>
<w:pPr>
<w:pStyle w:val="BodyText"/>
</w:pPr>
<w:r>
<w:t>
                        Todos los contenidos de los talleres están relacionados con el currículo de la enseñanza básica.
                        A diferencia de la práctica tradicional, pretendemos ahondar en el conocimiento partiendo de lo que realmente interesa al niño o niña,
                        ayudándole a que encuentre respuesta a las preguntas que él o ella se plantea.
                        
                        Por ese motivo, SaberMás proyecta estar al lado de los niños que necesitan una motivación extra para entender la escuela y fluir en ella,
                        y también al lado de aquellos a quienes la curiosidad y las ganas de saber les lleva más allá.
                    </w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>SaberMás trabaja con, desde y para la motivación</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Trabajamos en equipo en nuestros proyectos</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Cada uno va a su ritmo, y cada cual pone sus límites</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Aprendemos todos de todos</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Valoramos lo que hemos aprendido</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Estructura PBL: aprendemos cuando buscamos respuestas a nuestras propias preguntas</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Trabajo basado en la experimentación y en la investigación</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Alternativa a otras actividades de ocio</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Actividades centradas en el contexto cercano</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Duración limitada: 5-6 semanas</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Utilización de medios de expresión artística, digitales y analógicos</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Precio también limitado: 100-120?</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Uso de la tecnología durante todo el proceso de aprendizaje</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Niños y niñas que quieren saber más</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Flexibilidad en el uso de las lenguas de trabajo (inglés, castellano, esukara?)</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>De 8 a 12 años, sin separación por edades</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Máximo 10/1 por taller</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading1"/>
</w:pPr>
<w:r>
<w:t>Talleres temáticos</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Naturaleza</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Animales, Plantas, Piedras</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Arqueología</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Culturas Antiguas</w:t>
</w:r>
</w:p>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading3"/>
</w:pPr>
<w:r>
<w:t>Egipto, Grecia, China...</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Energía</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Paleontología</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Astronomía</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Arquitectura</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Cocina</w:t>
</w:r>
</w:p>
</wx:sub-section>
<wx:sub-section>
<w:p>
<w:pPr>
<w:pStyle w:val="Heading2"/>
</w:pPr>
<w:r>
<w:t>Poesía</w:t>
</w:r>
</w:p>
</wx:sub-section>
</wx:sub-section>
</wx:sub-section>
</wx:sect>
</w:body>
</w:wordDocument>
