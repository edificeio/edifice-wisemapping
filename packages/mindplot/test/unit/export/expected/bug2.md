# SaberMás

- Utilización de medios de expresión artística, digitales y analógicos
- Precio también limitado: 100-120?
- Talleres temáticos
	- Naturaleza
		- Animales, Plantas, Piedras
	- Arqueología
	- Culturas Antiguas
		- Egipto, Grecia, China...
	- Energía
	- Paleontología
	- Astronomía
	- Arquitectura
	- Cocina
	- Poesía
- Duración limitada: 5-6 semanas
- Niños y niñas que quieren saber más
- Alternativa a otras actividades de ocio
- Uso de la tecnología durante todo el proceso de aprendizaje
- Estructura PBL: aprendemos cuando buscamos respuestas a nuestras propias preguntas 
- Trabajo basado en la experimentación y en la investigación
- De 8 a 12 años, sin separación por edades
- Máximo 10/1 por taller
- Actividades centradas en el contexto cercano
- Flexibilidad en el uso de las lenguas de trabajo (inglés, castellano, esukara?)
- Complementamos el trabajo de la escuela[^1] 
	- SaberMás trabaja con, desde y para la motivación
	- Trabajamos en equipo en nuestros proyectos 
	- Cada uno va a su ritmo, y cada cual pone sus límites
	- Aprendemos todos de todos
	- Valoramos lo que hemos aprendido



[^1]: Todos los contenidos de los talleres están relacionados con el currículo de la enseñanza básica.A diferencia de la práctica tradicional, pretendemos ahondar en el conocimiento partiendo de lo que realmente interesa al niño o niña,
ayudándole a que encuentre respuesta a las preguntas que él o ella se plantea.

Por ese motivo, SaberMás proyecta estar al lado de los niños que necesitan una motivación extra para entender la escuela y fluir en ella,
 y también al lado de aquellos a quienes la curiosidad y las ganas de saber les lleva más allá.
