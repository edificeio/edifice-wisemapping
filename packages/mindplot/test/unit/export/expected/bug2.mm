<map version="1.0.1">
    <node ID="ID_1" TEXT="SaberMás">
        <node ID="ID_5" POSITION="right" STYLE="fork" TEXT="Utilización de medios de expresión artística, digitales y analógicos"/>
        <node ID="ID_9" POSITION="left" STYLE="fork" TEXT="Precio también limitado: 100-120?"/>
        <node ID="ID_2" POSITION="right" STYLE="fork" TEXT="Talleres temáticos">
            <node ID="ID_13" POSITION="right" STYLE="fork" TEXT="Naturaleza">
                <node ID="ID_17" POSITION="right" STYLE="fork" TEXT="Animales, Plantas, Piedras"/>
            </node>
            <node ID="ID_21" POSITION="right" STYLE="fork" TEXT="Arqueología"/>
            <node ID="ID_25" POSITION="right" STYLE="fork" TEXT="Culturas Antiguas">
                <node ID="ID_26" POSITION="right" STYLE="fork" TEXT="Egipto, Grecia, China..."/>
            </node>
            <node ID="ID_18" POSITION="right" STYLE="fork" TEXT="Energía"/>
            <node ID="ID_38" POSITION="right" STYLE="fork" TEXT="Paleontología"/>
            <node ID="ID_16" POSITION="right" STYLE="fork" TEXT="Astronomía"/>
            <node ID="ID_20" POSITION="right" STYLE="fork" TEXT="Arquitectura"/>
            <node ID="ID_11" POSITION="right" STYLE="fork" TEXT="Cocina"/>
            <node ID="ID_24" POSITION="right" STYLE="fork" TEXT="Poesía"/>
        </node>
        <node ID="ID_6" POSITION="left" STYLE="fork" TEXT="Duración limitada: 5-6 semanas"/>
        <node ID="ID_7" POSITION="left" STYLE="fork" TEXT="Niños y niñas que quieren saber más"/>
        <node ID="ID_8" POSITION="left" STYLE="fork" TEXT="Alternativa a otras actividades de ocio"/>
        <node ID="ID_23" POSITION="right" STYLE="fork" TEXT="Uso de la tecnología durante todo el proceso de aprendizaje"/>
        <node ID="ID_3" POSITION="right" STYLE="fork" TEXT="Estructura PBL: aprendemos cuando buscamos respuestas a nuestras propias preguntas "/>
        <node ID="ID_4" POSITION="right" STYLE="fork" TEXT="Trabajo basado en la experimentación y en la investigación"/>
        <node ID="ID_10" POSITION="left" STYLE="fork" TEXT="De 8 a 12 años, sin separación por edades"/>
        <node ID="ID_19" POSITION="left" STYLE="fork" TEXT="Máximo 10/1 por taller"/>
        <node ID="ID_37" POSITION="right" STYLE="fork" TEXT="Actividades centradas en el contexto cercano"/>
        <node ID="ID_22" POSITION="right" STYLE="fork" TEXT="Flexibilidad en el uso de las lenguas de trabajo (inglés, castellano, esukara?)"/>
        <node ID="ID_27" POSITION="right" STYLE="bubble" TEXT="Complementamos el trabajo de la escuela">
            <richcontent TYPE="NOTE">
                <html xmlns="http://www.w3.org/1999/xhtml">
                    <head></head>
                    <body>
                        <p>Todos los contenidos de los talleres están relacionados con el currículo de la enseñanza básica.</p>
                        <p>A diferencia de la práctica tradicional, pretendemos ahondar en el conocimiento partiendo de lo que realmente interesa al niño o niña,</p>
                        <p>ayudándole a que encuentre respuesta a las preguntas que él o ella se plantea.</p>
                        <p></p>
                        <p>Por ese motivo, SaberMás proyecta estar al lado de los niños que necesitan una motivación extra para entender la escuela y fluir en ella,</p>
                        <p>y también al lado de aquellos a quienes la curiosidad y las ganas de saber les lleva más allá.</p>
                    </body>
                </html>
            </richcontent>
            <node ID="ID_28" POSITION="right" TEXT="SaberMás trabaja con, desde y para la motivación"/>
            <node ID="ID_32" POSITION="right" STYLE="fork" TEXT="Trabajamos en equipo en nuestros proyectos "/>
            <node ID="ID_30" POSITION="right" STYLE="fork" TEXT="Cada uno va a su ritmo, y cada cual pone sus límites"/>
            <node ID="ID_31" POSITION="right" STYLE="fork" TEXT="Aprendemos todos de todos"/>
            <node ID="ID_33" POSITION="right" STYLE="fork" TEXT="Valoramos lo que hemos aprendido"/>
        </node>
    </node>
</map>