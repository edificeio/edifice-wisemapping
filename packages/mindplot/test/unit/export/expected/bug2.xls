<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String">SaberMás</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Complementamos el trabajo de la escuela</Data>
<Comment>
<ss:Data xmlns="http://www.w3.org/TR/REC-html40">
<p>Todos los contenidos de los talleres están relacionados con el currículo de la enseñanza básica.</p>
<p>A diferencia de la práctica tradicional, pretendemos ahondar en el conocimiento partiendo de lo que realmente interesa al niño o niña,</p>
<p>ayudándole a que encuentre respuesta a las preguntas que él o ella se plantea.</p>
<p/>
<p>Por ese motivo, SaberMás proyecta estar al lado de los niños que necesitan una motivación extra para entender la escuela y fluir en ella,</p>
<p>y también al lado de aquellos a quienes la curiosidad y las ganas de saber les lleva más allá.</p>
</ss:Data>
</Comment>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">SaberMás trabaja con, desde y para la motivación</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Trabajamos en equipo en nuestros proyectos </Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Cada uno va a su ritmo, y cada cual pone sus límites</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Aprendemos todos de todos</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Valoramos lo que hemos aprendido</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Estructura PBL: aprendemos cuando buscamos respuestas a nuestras propias preguntas </Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Trabajo basado en la experimentación y en la investigación</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Alternativa a otras actividades de ocio</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Actividades centradas en el contexto cercano</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Duración limitada: 5-6 semanas</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Utilización de medios de expresión artística, digitales y analógicos</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Precio también limitado: 100-120?</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Uso de la tecnología durante todo el proceso de aprendizaje</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Niños y niñas que quieren saber más</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Flexibilidad en el uso de las lenguas de trabajo (inglés, castellano, esukara?)</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">De 8 a 12 años, sin separación por edades</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Máximo 10/1 por taller</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Talleres temáticos</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Naturaleza</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Animales, Plantas, Piedras</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Arqueología</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Culturas Antiguas</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="4">
<Data ss:Type="String">Egipto, Grecia, China...</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Energía</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Paleontología</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Astronomía</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Arquitectura</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Cocina</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Poesía</Data>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
