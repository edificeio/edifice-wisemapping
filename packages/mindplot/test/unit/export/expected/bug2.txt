1 SaberMás
	1.1 Utilización de medios de expresión artística, digitales y analógicos
	1.2 Precio también limitado: 100-120?
	1.3 Talleres temáticos
		1.3.1 Naturaleza
			1.3.1.1 Animales, Plantas, Piedras
		1.3.2 Arqueología
		1.3.3 Culturas Antiguas
			1.3.3.1 Egipto, Grecia, China...
		1.3.4 Energía
		1.3.5 Paleontología
		1.3.6 Astronomía
		1.3.7 Arquitectura
		1.3.8 Cocina
		1.3.9 Poesía
	1.4 Duración limitada: 5-6 semanas
	1.5 Niños y niñas que quieren saber más
	1.6 Alternativa a otras actividades de ocio
	1.7 Uso de la tecnología durante todo el proceso de aprendizaje
	1.8 Estructura PBL: aprendemos cuando buscamos respuestas a nuestras propias preguntas 
	1.9 Trabajo basado en la experimentación y en la investigación
	1.10 De 8 a 12 años, sin separación por edades
	1.11 Máximo 10/1 por taller
	1.12 Actividades centradas en el contexto cercano
	1.13 Flexibilidad en el uso de las lenguas de trabajo (inglés, castellano, esukara?)
	1.14 Complementamos el trabajo de la escuela
	  [Note: Todos los contenidos de los talleres están relacionados con el currículo de la enseñanza básica.
A diferencia de la práctica tradicional, pretendemos ahondar en el conocimiento partiendo de lo que realmente interesa al niño o niña,
ayudándole a que encuentre respuesta a las preguntas que él o ella se plantea.

Por ese motivo, SaberMás proyecta estar al lado de los niños que necesitan una motivación extra para entender la escuela y fluir en ella,
 y también al lado de aquellos a quienes la curiosidad y las ganas de saber les lleva más allá.]
		1.14.1 SaberMás trabaja con, desde y para la motivación
		1.14.2 Trabajamos en equipo en nuestros proyectos 
		1.14.3 Cada uno va a su ritmo, y cada cual pone sus límites
		1.14.4 Aprendemos todos de todos
		1.14.5 Valoramos lo que hemos aprendido
